### ISSUE TITLE
Add o1-preview and o1-mini

### ISSUE THREAD

**Issue by josephfinlayson** [mckaywrigley/chatbot-ui#1860]
It's out for some users. Can you add?

https://platform.openai.com/docs/models/o1

**Comment by Bortus-AI** [comment]
Yes that would be great. I've had access for a week and had to change to LobeChat since they have it supported already

**Comment by AndrewEAUS** [comment]
Anyone have any luck with this? O1 has different requirements for example no system message yet.

**Comment by spammenotinoz** [comment]
Yes, just added via litellm and before that open router.

On 27 Oct 2024, at 1:59 pm, AndrewEAUS ***@***.***> wrote:

﻿

Anyone have any luck with this? O1 has different requirements for example no system message yet.

**Comment by DiamondGlassDrill** [comment]
O1 is available now for Tier 5 users, has someone added it already into Chatbot UI?
PS System Messages are as well available now, as is reasoning length.

**Comment by Tost-1** [comment]
It seems o1-Preview still doesn't show up, has anyone else found a way to get it working?
